backend/annotation/models.py

```diff
@@ -42,8 +42,11 @@ def annotation_object(self):
         obj['IRI'] = self.IRI
         obj['motivation'] = self.motivation
         obj['creator'] = self.creator
-        target = Target.objects.get(annotation=self)
-        obj['target'] = target_json(target)
+        targets = Target.objects.filter(IRI=self.IRI)
```

Comment:
Please use the previous one. There may be multiple targets for the same url but they may be related to different annotations